tmeister/static/index.html

```diff
@@ -17,8 +13,6 @@
 	<body id="cps-app">
 		<div id="main-content">
 		</div>
-		<script type="text/javascript">
-			SystemJS.import('build/toggle-meister.js');
-		</script>
+    <script type="text/javascript" src="build/toggle-meiser.js"></script></body>
```

Comment:
same minor typo here. Missing `t`

Reply:
That's why I didn't notice I had the same typo twice!! :grin: